keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -779,6 +783,7 @@ https://storage.googleapis.com/k8s-triage/index.html
 -->
 
 - <test>: <link to test coverage>
+SIG node does not typically write e2e tests
```

Comment:
```suggestion
SIG node does not typically write integration tests
```

TBH I would probably expand on this. We don't write integration tests because we kubelet is covered by the e2e_node suite. We as a sig would love to write more integration tests but we are limiting in what we can do.

Reply:
```suggestion
Integration tests for the kubelet are not existing yet and will be covered by extending the available e2e test suites.
```

Reply:
Sascha's look more correct, this is not about what one SIG does, is just because this feature does not require them or are covered with e2e ... if SIG node has to write any API object related feature must probably it must require integration tests.

Reply:
> We as a sig would love to write more integration tests but we are limiting in what we can do.

Careful here: if lack of resources is the reason for not writing integration tests when they would be desirable for a feature, then development of that feature needs to slow down. Please don't skip testing to deliver features faster.

Reply:
sig-node's difficulty with e2e tests has historically been that e2e tests don't give control of the kubelet flags.  this feature appears to be functional without changing kubelet flags, so why can't we write an e2e test for this?